app/views/visitor_mailer/booked.html.erb

```diff
@@ -97,3 +97,10 @@
 <p>
   <small><%= t(".visit_id") %> <%= @visit.id %></small>
 </p>
+</p>
+<div class="apvs_survey">
+   <p><strong><%= t('.survey_question1') %></strong></p>
```

Comment:
The locales wont work because it will try to find the text under the `booked` section of the locales. I think this should work:

```ruby
t('survey_question1', scope: %i(visitor_mailer request_acknowledged))
```